docs/API.md

```diff
@@ -49,5 +52,18 @@ For example: Accessing the data for a chat report will return the data if the re
 - Nouns can be optional eg. `SignIn`
 - Verbs should be unique and indicate the user's action (as perceived by the user). eg. `Request`, `Open`, `Accept`, `Pay`
   - If a unique verb cannot be used, then use only the standard verbs: `Update`, `Add`, `Delete`
+  - Use Create/Delete to create or destroy the only copy of data
+  - Use Add/Remove when we're just dealing with the copy of data
 - Names must NOT include names of parameters eg. Bad: `SignInWithPasswordOr2FA` Good: `SignIn`
+- Names must NOT the word Page or View, eg. Bad: `OpenReportPage` Good: `OpenReport`
 - Names must NOT reveal backend implementation details that the user does not know about eg. `AddVBBA` (users don't know what a VBBA is - verified business bank account) Good: `AddBankAccount`
+- For deep links like `StartAppWhileSignedInAndOpenWorkspace`, create separate commands for each specific scenario like `StartAppWhileSignedIn` and `OpenWorkspace`, where both requests will be triggered in parallel.
+
+## FAQ
+Q: How should error messages be persisted to Onyx?
+A:- The API layer Pusher onyxData should set specific error messages.
+  - In cases where the API does not set an error, a generic error message can be set in the API.write onyxData `failureData` block.
+  - Previous errors should be cleared by local onyxData when a new request is made
```

Comment:
https://expensify.slack.com/archives/C02HWMSMZEC/p1654882317724299

Reply:
Oh that's right, I'm going to replace the FAQ formatting with another heading level (using ###)